app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    issues_found = 0
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            issues_found += 1
+            return "All friends should be vaccinated"
```

Comment:
Returning immediately after catching a `VaccineError` means the function will exit on the first unvaccinated friend, potentially missing other issues. Consider collecting all issues first and returning a summary after the loop.